src/ggplotnim.nim

```diff
@@ -3,7 +3,7 @@
 import sequtils, tables, sets, algorithm, strutils, macros
 import parsecsv, streams, hashes, sugar, math
 
-when (NimMajor, NimMinor, NimPatch) > (1, 2, 0):
+when (NimMajor, NimMinor, NimPatch) > (1, 3, 0):
```

Comment:
```suggestion
when (NimMajor, NimMinor, NimPatch) >= (1, 3, 0):
```

Shouldn't it be this then? I forgot about patch releases for 1.2 (sorry), but devel should still continue to work.

Reply:
Devel started with 1.3.1 (same with 1.1.1), and it is always odd patch version because of some internal logic of determining (non) stable versions.

Reply:
Oh, I had no idea. Thanks!